packages/gatsby-cli/src/index.ts

```diff
@@ -17,17 +17,28 @@ if (useJsonLogger) {
 // Check if update is available
 updateNotifier({ pkg }).notify({ isGlobal: true })
 
-const MIN_NODE_VERSION = `>=8.0.0`
+const MIN_NODE_VERSION = `8.0.0`
+const NEXT_MIN_NODE_VERSION = `10.13.0`
 
-if (!semver.satisfies(process.version, MIN_NODE_VERSION)) {
+if (!semver.satisfies(process.version, `>=${MIN_NODE_VERSION}`)) {
   report.panic(
     report.stripIndent(`
-      Gatsby requires Node.js v8 or higher (you have ${process.version}).
+      Gatsby requires Node.js ${MIN_NODE_VERSION} or higher (you have ${process.version}).
       Upgrade Node to the latest stable release: https://gatsby.dev/upgrading-node-js
     `)
   )
 }
 
+if (!semver.satisfies(process.version, `>=${NEXT_MIN_NODE_VERSION}`)) {
+  report.warn(
+    report.stripIndent(`
+      Node.js ${process.version} has reached End of Life status on 31 December, 2019. 
+      Gatsby will only actively support ${NEXT_MIN_NODE_VERSION} or higher and drop support for Node 8 soon.
```

Comment:
maybe remove here the version number so on the next time we don't need to change the numbers... 

```suggestion
      Gatsby will only actively support ${NEXT_MIN_NODE_VERSION} or higher and drop support for earlier Node version soon.
```